app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class=" vh-100 bg-image">
+ <div class="mask d-flex align-items-center h-100 gradient-custom-3">
+    <div class="container h-100">
+      <div class="row d-flex justify-content-center align-items-center h-100">
+        <div class="col-12 col-md-9 col-lg-7 col-xl-6">
+          <div class="card" style="border-radius: 15px;">
+            <div class="card-body p-5">
+              <h2 class="text-uppercase text-center mb-5">Create an account</h2>
+<div class="form-submit">
+  <%= form_for(resource, as: resource_name, html: {'data-turbo' => "false"}, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+    <div class="field">
+      <%= f.label :Name %><br />
+      <%= f.text_field :Name, autocomplete: "name" %>
+    </div>
+    <div class="field">
+      <%= f.label :Photo %><br />
+      <%= f.text_field :Photo, autocomplete: "photo" %>
+    </div>
+    <div class="field">
+      <%= f.label :Bio %><br />
+      <%= f.text_area :Bio, autocomplete: "bio" %>
+    </div>
+    <%= f.hidden_field :PostsCounter, value: 0 %>
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
```

Comment:
- [ ] It will be great to follow the given wireframe design for the register screen. This is a major requirement for this project. If you can check the wireframe provided, You will notice that your register screen doesn't resemble it. Kindly improve on this.

Reply:
they didn't give us registration wireframe ,so I think you can customise how you want

Reply:
Hello, try and make it resemble  the login  wireflame  just for consistency. But yes I agree with you they didn't  provide the registration  wire frame I will let them know

Reply:
fixed login screen